ui/components/ui/custom-nonce/custom-nonce.js

```diff
@@ -0,0 +1,79 @@
+import React, { useContext } from 'react';
```

Comment:
I think this should go in `ui/components/app`

Reply:
@jpuri Done [here](https://github.com/MetaMask/metamask-extension/pull/17945/commits/c493916a34f2ff6cb424b823f64cd5f0bad9385b).